lib/bundler/cli.rb

```diff
@@ -522,23 +522,21 @@ def licenses
       end
     end
 
-    if Bundler.feature_flag.viz_command?
```

Comment:
I think this line should actually be restored, I'll try reverting these changes.